perf/lua/benchmark.lua

```diff
@@ -0,0 +1,106 @@
+-- Usage:
+--
+-- local benchmark = require('benchmark')
+-- local clock = require('clock')
+--
+-- These options are parsed by the module by default:
+-- -- output = 'string',
+-- -- output_format = 'string',
+-- local opts = benchmark.argparse(arg, {
+--     <..your options..>
+-- })
+-- benchmark.new(
+--     require('internal.argparse').parse(opts)
+-- )
```

Comment:
`benchmark.argparse` and `argparse.parse` are called both in this example. It seems, it is by a mistake.

Reply:
Fixed, thanks!